modules/ROOT/pages/restarting-a-node.adoc

```diff
@@ -3,76 +3,112 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic describes the processes for removing an Anypoint Platform PCE node from active service, and returning it to service when ready.
+== Before You Begin
 
-== Notes
+* Set up and test the backup procedure on all clusters after you complete installation.
+* Perform a backup before attempting node maintenance to preserve data in the event of a failure.
 
-[NOTE]
-To preserve your data in case of system failure, ensure that your system is backed up before attempting node maintenance.
-[NOTE]
-The steps below must be performed on a single Anypoint Platform PCE node at a time.  If maintenance is required on more than one node, the entire process must be completed on a single node before the next node is addressed.
+== Cluster and Application Updates
 
-== Drain the Node
+The following steps must be performed on one Anypoint Platform Private Cloud Edition node at a time to minimize or eliminate component breakage during the update process. The entire process must be completed on each node before the next node is addressed. Verify the health status of all cluster components and applications after each node update before proceeding with the next node.
 
-Before any maintenance or restart of a node can be performed, it must be removed from the cluster and its workload reassigned to the other nodes in the cluster.
+This approach assumes that multiple masters are running.
```

Comment:
we can remove this past of the `masters` as it doesn't add yo our customers.

Reply:
I didn't write the new version. So I'll leave it as it is as I guess this request is not for me.

Reply:
@paigehargrave that comment from @federicoboerr is probably for you.

Reply:
I removed the line - thanks!